### ISSUE TITLE
WordPress 6.5 – Data Views accessibility audit

### ISSUE THREAD

**Issue by andrewhayward** [WordPress/gutenberg#59643]
With [the imminent release of WordPress 6.5](https://wordpress.org/news/2024/03/wordpress-6-5-release-candidate-1/), the new admin views will be publicly available for the first time. This issue tracks accessibility issues within that scope, at that point of release. It is not necessarily intended as a full and in-depth list of every minor problem, but rather to address the more obvious issues and gauge priorities for the 6.6 release.

## In progress

This issue will be updated as problems are identified.

**Comment by carolinan** [comment]
@andrewhayward  HI, is there an update on this task?

Page Patterns:
I am not able to access the pattern description nor the context that "Theme & plugin patterns cannot be edited." 
Both texts are printed in div's that use the `hidden` attribute, and both have a unique id.

The id's seems to be intended to be used in an `aria-describedby` in a component called `<PreviewWrapper>` which is not  printed in the markup for the type of patterns that the context is important for.

Related code:
https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-site/src/components/page-patterns/index.js#L145
https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-site/src/components/page-patterns/index.js#L168
https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-site/src/components/page-patterns/index.js#L109

**Comment by carolinan** [comment]
@andrewhayward It would be good to proceed with the accessibility audit, are you able to dedicate the time needed?

**Comment by ntsekouras** [comment]
Closing for now as this hasn't been updated for a while and we have separate a11y issues, when are found.
